src/components/Navigation/NavigationTabBar/NavigationTabBarDummy.tsx

```diff
@@ -0,0 +1,297 @@
+                </View>
+                <Text
+                    style={[
+                        styles.textSmall,
+                        styles.textAlignCenter,
+                        styles.mt1Half,
+                        selectedTab === NAVIGATION_TABS.SEARCH ? styles.textBold : styles.textSupporting,
+                        styles.navigationTabBarLabel,
+                    ]}
+                >
+                    {translate('common.reports')}
+                </Text>
+            </PressableWithFeedback>
+            <View style={[styles.flex1, styles.navigationTabBarItem]}>
+                <FloatingActionButton
+                    onPress={noop}
+                    isActive={false}
+                    accessibilityLabel={translate('sidebarScreen.fabNewChatExplained')}
+                    role={CONST.ROLE.BUTTON}
+                    isTooltipAllowed={false}
+                />
+            </View>
+            <PressableWithFeedback
+                onPress={noop}
+                role={CONST.ROLE.BUTTON}
+                accessibilityLabel={translate('common.workspacesTabTitle')}
+                wrapperStyle={styles.flex1}
+                style={styles.navigationTabBarItem}
+            >
+                <View>
+                    <Icon
+                        src={Expensicons.Buildings}
+                        fill={selectedTab === NAVIGATION_TABS.WORKSPACES ? theme.iconMenu : theme.icon}
+                        width={variables.iconBottomBar}
+                        height={variables.iconBottomBar}
+                    />
+                </View>
+                <Text
+                    style={[
+                        styles.textSmall,
+                        styles.textAlignCenter,
+                        styles.mt1Half,
+                        selectedTab === NAVIGATION_TABS.WORKSPACES ? styles.textBold : styles.textSupporting,
+                        styles.navigationTabBarLabel,
+                    ]}
+                >
+                    {translate('common.workspacesTabTitle')}
+                </Text>
+            </PressableWithFeedback>
+            <NavigationTabBarAvatar
+                style={styles.navigationTabBarItem}
+                isSelected={selectedTab === NAVIGATION_TABS.SETTINGS}
+                onPress={noop}
+            />
+        </View>
+    );
+}
+
+/**
+ * This is a dummy component for the NavigationTabBar created for performance reasons.
+ * It is used to render the dummy NavigationTabBar in a wide or narrow layout.
+ * @param selectedTab - The selected tab.
+ * @returns The NavigationTabBarDummy component.
+ */
+function NavigationTabBarDummy({selectedTab}: {selectedTab: ValueOf<typeof NAVIGATION_TABS>}) {
+    const {shouldUseNarrowLayout} = useResponsiveLayout();
+
+    if (shouldUseNarrowLayout) {
+        return <NavigationTabBarNarrowDummy selectedTab={selectedTab} />;
+    }
+
+    return <NavigationTabBarWideDummy selectedTab={selectedTab} />;
```

Comment:
In which case do we need this in the Wide layout? Could we not render this at all in wide?